docs/pages/switch.js

```diff
@@ -57,6 +57,196 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+        </Label>
+      </Flex.Item>
+      <Switch
+        onChange={() => setSwitched(!switched)}
+        id="makesecret"
+        switched={switched}
+      />
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Truncate label text, instead allow it to wrap to form another line. "
+            defaultCode={`
+function SwitchExample3() {
+  const [switched, setSwitched] = React.useState(false);
+
+  return (
+    <Flex gap={2} alignItems="center">
+      <Switch
+        onChange={() => setSwitched(!switched)}
+        id="setboard"
+        switched={switched}
+      />
+      <Label htmlFor="setboard">
+        <Text>Set board to...</Text>
+      </Label>
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Communicate why a switch is disabled and how to enable it if possible."
+            defaultCode={`
+<Flex gap={2} alignItems="center">
+  <Switch
+    onChange={() => {}}
+    id="showsecretboards"
+    switched={false}
+    disabled={true}
+  />
+  <Label htmlFor="showsecretboards">
+    <Flex direction="column" gap={1}>
+      <Text color="subtle">Show secret boards</Text>
+      <Text size="100" color="subtle">You don't have any secret boards</Text>
+    </Flex>
+  </Label>
+</Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use alternative styling to represent the functionality of a switch. Use Switch instead."
+            defaultCode={`
+<Flex direction="column" gap={2} width="100%" maxWidth={300}>
+  <Heading size="300" accessibilityLevel={3}>On Pinterest notifications</Heading>
+  <Flex direction="column" gap={4}>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Comments</Text>
+      </Flex.Item>
+      <Icon icon="check-circle" color="default" accessibilityLabel="checked circle" />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Mentions</Text>
+      </Flex.Item>
+      <Icon icon="check-circle" color="default" accessibilityLabel="checked circle" />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Views</Text>
+      </Flex.Item>
+      <Icon icon="circle-outline" accessibilityLabel="unchecked circle" />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Saves</Text>
+      </Flex.Item>
+      <Icon icon="check-circle" color="default" accessibilityLabel="checked circle" />
+    </Flex>
+  </Flex>
+</Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use a standalone switch if the surrounding context is clear without an associated label. However, be sure to include a visually hidden label for screen readers."
```

Comment:
Keep in mind that the code blocks aren't available for Dos/Don'ts, so the reader won't be able to tell that this is implementing a separate Label. There's nothing visually indicating that this is using Label instead of `label`, so this doesn't really communicate anything. Given that we mention this in the Accessibility section and the "With a label" variant, this Don't may not be necessary?

Reply:
maybe just convert this example to the "With a label example"

Reply:
I think think will be helpful from a design perspective. I'd argue to keep.

Reply:
works for me!

Reply:
> maybe just convert this example to the "With a label example"

Sorry, @ayeshakmaz, could you elaborate?

Reply:
oh I meant instead of having this example as a best practice, use this code for the "with a label" variant example

Reply:
If you think it's self-explanatory for designers, 👍

Reply:
Oh, nice! I like it! Done!

Reply:
Yeah, @dangerismycat, let's see what we hear and adjust if necessary. @khalvy, just wanted to confirm if you're cool with this change.